src/components/PoolCreator/Wizard/Wizard.js

```diff
@@ -166,14 +166,20 @@ export const Wizard = (props) => {
 
     // create pool contract on the blockchain
     // Get blockchain data after tx confirms, then update pool object
+
+    const args = [
+      config.network.addresses.cDai,
+      config.network.operator,
```

Comment:
what is operator? can you give it a descriptive name?

Reply:
operator is our backend account that is responsible to call the oracle in case there was a winner